### DISCUSSION TITLE
List of FX names not matching up

### DISCUSSION THREAD

**Discussion by azarius74** [Feu-Secret/Tokenmagic#118] [Q&A] [answered]
Greetings!  First off love the module you've created...  Do you have a Patreon for it?

I've noticed that the names of the spells don't always match up with the names from the drop down menus... IE:  In one, it says "Magearmour 01", but there is no Magearmour" at all in the drop down menu.  Have I missed something simple?

Cheers

**Comment by Feu-Secret** [comment, marked as answer]
> 
> 
> Greetings! First off love the module you've created... Do you have a Patreon for it?
> 
Hello and thank you! I don't have a Patreon for the moment. 

> I've noticed that the names of the spells don't always match up with the names from the drop down menus... IE: In one, it says >"Magearmour 01", but there is no Magearmour" at all in the drop down menu. Have I missed something simple?
>
Could you publish a screenshot please? I can't identify the drop down menu you're talking about. It may be a module that uses Token Magic.

Cheers
